backend/controller/admin/admin.go

```diff
@@ -207,3 +225,50 @@ func (s *AdminService) SecretUnset(ctx context.Context, req *connect.Request[ftl
 	}
 	return connect.NewResponse(&ftlv1.UnsetSecretResponse{}), nil
 }
+
+func (s *AdminService) validateAgainstSchema(ctx context.Context, isSecret bool, ref cf.Ref, value json.RawMessage) error {
+	sch, err := s.schr.GetActiveSchema(ctx)
```

Comment:
We can't validate a global, so we may as well early exit here. We should come up with a solution for that, can you file a ticket please?